extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -36,7 +35,6 @@ import {toggle} from '../../../src/style';
 installActivityService(AMP.win);
 installCidService(AMP.win);
 installCryptoService(AMP.win);
-installStorageService(AMP.win);
```

Comment:
Good catch. Should likely double install.

On Tue, Sep 13, 2016 at 4:39 PM, Dima Voytenko <notifications@github.com>
wrote:

> In extensions/amp-analytics/0.1/amp-analytics.js
> <https://github.com/ampproject/amphtml/pull/4944#discussion_r78664777>:
>
> > @@ -36,7 +35,6 @@ import {toggle} from '../../../src/style';
> >  installActivityService(AMP.win);
> >  installCidService(AMP.win);
> >  installCryptoService(AMP.win);
> > -installStorageService(AMP.win);
>
> We gonna get a bit of a version skew. @cramforce
> <https://github.com/cramforce> is this ok? Or should we do
> double-installation for one cycle?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ampproject/amphtml/pull/4944/files/38ae18a72ba2f5a6a8e5dbfb212ee19888c057fa#r78664777>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFeT7T7I6JlXDfQO7vR-0RmLGOLYGFjks5qpzQvgaJpZM4J7GGg>
> .
>